recipes/pdb2pqr_htmd_propka30/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pdb2pqr_htmd_propka30" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 11038a630e31c823ea3cfe3d9f878e18edd4517b6a8916cb3e0127e1fa96035e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - pdb2pqr_cli = pdb2pqr.main:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - pdb2pqr
+    - pdb2pqr.extensions
+    - pdb2pqr.propka30
+    - pdb2pqr.propka30.Source
+    - pdb2pqr.src
+  commands:
+    - pdb2pqr_cli  --ff CHARMM --ffout CHARMM --chain --ph-calc-method=propka $SP_DIR/pdb2pqr/test/1rxz.pdb  tmp.pdb
```

Comment:
I'm wondering if this is failing on windows because of the path. Can you just make this something like `pdb2pqr_cli --help`, just to check that the command line tool is there, rather than testing it's actual functionality?

Reply:
It's done.
Thanks for the suggestion !

Reply:
Thanks for the tips.
It is solved now.